Rename trpSSR into trpcSSR

fatjan opened the pull request:
Closes #8 

## Description

Rename all `trpSSR` variables into `trpcSSR`.

Commit:
style(rename variable): rename trpSSR into trpcSSR

zakiego commented:
Seems like one thing remains unchanged, everything else is perfect

![image](https://github.com/zakiego/desa-bambangin-2023/assets/78015359/0a7c6faf-0a7f-4eaf-be72-6b971d7deb15)

Commit:
tyle(rename variable): rename trpSSR to trpcSSR

fatjan commented:
> Seems like one thing remains unchanged, everything else is perfect
> 
> ![image](https://user-images.githubusercontent.com/78015359/251037314-0a7c6faf-0a7f-4eaf-be72-6b971d7deb15.png)

Thanks for checking, I have added a new commit but I can't see the new commit in this PR.
My [branch](https://github.com/zakiego/desa-bambangin-2023/compare/main...fatjan:desa-bambangin-2023:rename-trpSSR-to-trpcSSR) has the two commits though.
Could you please suggest on how I should fix this?
Should I close this PR and create a new one?

Updated: I have synced the repo and now this PR has the latest commit to rename the variable. Kindly review the PR again. Thank you.

fatjan committed:
Merge branch 'zakiego:main' into rename-trpSSR-to-trpcSSR

zakiego commented:
Perfect. Thank you @fatjan! 🎉
